modules/core/04-channel/keeper/keeper.go

```diff
@@ -539,6 +539,18 @@ func (k Keeper) SetUpgrade(ctx sdk.Context, portID, channelID string, upgrade ty
 	store.Set(host.ChannelUpgradeKey(portID, channelID), bz)
 }
 
+// deleteCounterpartyLastPacketSequence deletes the counterparty last packet sequence from the store.
+func (k Keeper) deleteCounterpartyLastPacketSequence(ctx sdk.Context, portID, channelID string) {
```

Comment:
taken from #3728 temporarily.

Reply:
Maybe we should open a quick pr to merge these changes?

Reply:
good idea https://github.com/cosmos/ibc-go/pull/3780